packages/turf-along/tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
   "extends": "../../tsconfig.shared.json",
   "compilerOptions": {
-    "outDir": "dist/js"
+    "allowJs": true
```

Comment:
@RobinVdBroeck are these changes the same for all tsconfig.json files? If so, can we push them into the tsconfig.shared.json file and simplify these to just extend that?